pkgs/development/libraries/libraw/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, autoreconfHook, lcms2, pkg-config }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Comment:
rec?

Reply:
Ah right.  The first commit needed that to inherit version, but it's not needed anymore after the second commit.

Reply:
So amended to drop that.